src/main/java/org/myrobotlab/service/InMoovHead.java

```diff
@@ -102,12 +79,7 @@ public boolean enable() {
   @Override
   public void broadcastState() {
     // notify the gui
-    rothead.broadcastState();
-    rollNeck.broadcastState();
-    neck.broadcastState();
-    eyeX.broadcastState();
-    eyeY.broadcastState();
-    jaw.broadcastState();
+    Runtime.broadcastStates();    

```

Comment:
same comment as above.. why does this need to be overridden here?  seems like the base class should handle this method.